lib/Model/CommandMapper.php

```diff
@@ -85,8 +85,13 @@ public function find(string $app, string $cmd): Command {
 		$query = $this->db->getQueryBuilder();
 		$query->select('*')
 			->from($this->getTableName())
-			->where($query->expr()->eq('app', $query->createNamedParameter($app)))
-			->andWhere($query->expr()->eq('command', $query->createNamedParameter($cmd)));
+			->where($query->expr()->eq('command', $query->createNamedParameter($cmd)));
```

Comment:
should the select * above be replaced as well for Oracle ?

Reply:
No, it's only one table being queried, so it's fine.